news/2021-12-27-clickers-choice-2021.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Clicker's Choice: Top 20 osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+Welcome aboard a brand new platform to help you vote for the top 20 osu! players of 2021!
```

Comment:
Probably change this line to read less liek this is an official project..

Reply:
Drawing a total blank on wording for this one. Very open to suggestions.

Reply:
Had a long think on this one - brainstorming sure can be tedious sometimes. Maybe some combination of these will work.

```suggestion
Take part in helping the community determine the top 20 osu! players of 2021!
```

```suggestion
Come and chip in with your take on who were the best osu! players of 2021!
```

```suggestion
It's time for everyone to decide on who were the best osu! players of 2021!
```

```suggestion
Take part in helping everyone collectively identify the best osu! players of 2021!
```

```suggestion
Come on in and vote for your favorite circle clickers of the past year!
```